File: backend/src/main/java/com/satc/integrador/ai/exercicios/ExercicioVocabualrioParesGetDto.java
package com.satc.integrador.ai.exercicios;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ExercicioVocabualrioParesGetDto {
    private Integer id;
    private Integer idOrdemExercicio;
    private Integer idPlanoEstudo;

    private List<String> paresEsquerda;
    private List<String> paresDireita;
}

File: android/app/src/main/java/com/satc/integrador_ai/data/model/request/ExercicioVocabualrioParesGetDto.kt
package com.satc.integrador_ai.data.model.request

import com.satc.integrador_ai.storage.TipoExercicio

class ExercicioVocabualrioParesGetDto (
    val id: Int ?= null,
    val idOrdemExercicio: Int ?= null,
    val idPlanoEstudo: Int ?= null,
    val paresEsquerda: MutableList<String> ?= null,
    val paresDireita: MutableList<String> ?= null
) : TipoExercicio {
    override fun getTipoExercicio(): String {
        return "VocabularioPares"
    }
}